website/docs/api/bigint-money/createBigIntMoney/allocate.md

```diff
@@ -5,7 +5,15 @@ hide_title: true
 sidebar_label: allocate
 ---
 
-# allocate
+# `allocate(ratios)`
+
+#### Arguments
+
+1. `ratios (Array<number>)`
+
+#### Returns
+
+`Array` ([BigIntMoneyBase](Description.md#bigintmoneybase))
```

Comment:
Could you make here something similar to example below? Or it isn't working properly

returns `Array<([BigIntMoneyBase](Description.md#bigintmoneybase))>`